src/main/java/net/minecraftforge/common/capabilities/CapabilityManager.java

```diff
@@ -49,7 +49,10 @@
      * @param type The Interface to be registered
      * @param storage A default implementation of the storage handler.
      * @param implementation A default implementation of the interface.
+     * @deprecated Pass a constructor reference instead of a class name as the third parameter
```

Comment:
I'd just mention to use the `register` method that takes an explicit factory. There is no requirement to use a constructor reference, although that is a nice way to do it.